using-timescaledb/ingesting-data.md

```diff
@@ -0,0 +1,24 @@
+# Ingesting data
+
+TimescaleDB can support standard SQL inserts. Read more about how to use
+SQL to write data into TimescaleDB in our [Writing Data][writing-data] section.
+
+Users often choose to leverage existing 3rd party tools to build data ingest
+pipelines. Any tools that work with PostgreSQL also work with TimescaleDB.
+
+## Prometheus [](prometheus)
+
+Prometheus is a popular tool used to monitor infrastructure metrics. It can scrape
+any endpoints that expose metrics in a Prometheus-compatible format. Prometheus
+can use TimescaleDB as a remote storage solution to store long-term metrics. By
```

Comment:
Avoid the use of "by" twice?  "Combine TimescaleDB with Prometheus to expand your..."

Reply:
Done!